.gitignore

```diff
@@ -1,2 +1,5 @@
 # IDE-specific metadata
 .idea/
+
+# Environment variables. Useful for examples.
+.env
```

Comment:
Lack of new line at end of file is inconsistent with previous format and may cause a "false positive diff" to appear if someone were to append to this file in the future.

Reply:
I will fix it.